src/cosmosdb-preview/azext_cosmosdb_preview/_params.py

```diff
@@ -142,3 +142,53 @@ def load_arguments(self, _):
         c.argument('role_definition_name', options_list=['--role-definition-name', '-n'], help="Unique Name of the Role Definition that this Role Assignment refers to. Eg. 'Contoso Reader Role'.")
         c.argument('scope', validator=validate_scope, options_list=['--scope', '-s'], help="Data plane resource path at which this Role Assignment is being granted.")
         c.argument('principal_id', options_list=['--principal-id', '-p'], help="AAD Object ID of the principal to which this Role Assignment is being granted.")
+
+    # Managed Cassandra Cluster
+    with self.argument_context('cassandra-managed-instance cluster create') as c:
+        c.argument('location', options_list=['--location', '-l'], help="Azure Location of the Cluster", required=True)
+        c.argument('cluster_name', options_list=['--cluster-name', '-c'], help="Cluster Name", required=True)
+        c.argument('delegated_management_subnet_id', options_list=['--delegated-management-subnet-id', '-s'], help="The subnet we will inject the Network Interface of the component which orchestrates the cassandra cluster. This is also where customers can access the prometheus endpoint", required=True)
+
+    # Managed Cassandra Cluster
+    with self.argument_context('cassandra-managed-instance cluster update') as c:
+        c.argument('cluster_name', options_list=['--cluster-name', '-c'], help="Cluster Name", required=True)
+
+    # Managed Cassandra Cluster
+    with self.argument_context('cassandra-managed-instance cluster node-status') as c:
+        c.argument('cluster_name', options_list=['--cluster-name', '-c'], help="Cluster Name", required=True)
+
+    # Managed Cassandra Cluster
+    with self.argument_context('cassandra-managed-instance cluster show') as c:
+        c.argument('cluster_name', options_list=['--cluster-name', '-c'], help="Cluster Name", required=True)
+
+    # Managed Cassandra Cluster
+    with self.argument_context('cassandra-managed-instance cluster delete') as c:
+        c.argument('cluster_name', options_list=['--cluster-name', '-c'], help="Cluster Name", required=True)
```

Comment:
Actually you can create an `cluster_name_type` and apply it to the whole `cluster` command group to avoid duplications.
```suggestion
    from knack.arguments import CLIArgumentType
    cluster_name_type = CLIArgumentType(options_list=['--cluster-name', '-c'], help="Cluster Name", required=True)
    # Managed Cassandra Cluster
    with self.argument_context('cassandra-managed-instance cluster') as c:
        c.argument('cluster_name', cluster_name_type)
```